webapp/src/modules/mapBuilder.js

```diff
@@ -31,11 +31,9 @@ export default class MapBuilder {
     this.newLocateControl().addTo(this._map);
     this._lastExtendedBounds = expandBounds(this._map.getBounds());
     this._mcg = this._l.markerClusterGroup({
+      spiderfyOnMaxZoom: false,
       showCoverageOnHover: false,
-      maxClusterRadius: (mapZoom) => {
-        // only cluster same location markers above zoom 17
-        return mapZoom >= 17 ? 0 : 32;
-      },
+      maxClusterRadius: 32,
```

Comment:
We cluster all vehicles at any zoom level now since these changes will omit "spiderfying" any same-location vehicles.